concepts/05 Widgets/Form/01 Getting Started with Form/40 Organize the Form Items/10 In Columns.md

```diff
@@ -0,0 +1,163 @@
+You can divide form items into several columns. To keep the fixed number of columns, specify the number in the [colCount](/Documentation/ApiReference/UI_Widgets/dxForm/Configuration/#colCount) property. An item can occupy more than one column. For this, specify the [colSpan](/Documentation/ApiReference/UI_Widgets/dxForm/Item_Types/SimpleItem/#colSpan) property. In the example below, the `Notes` item spans two columns:
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#formContainer").dxForm({
+            formData: {
+                name: "John Heart",
+                position: "CEO",
+                hireDate: new Date(2012, 4, 13),
+                officeNumber: 901,
+                notes: "John has been in the Audio/Video industry since 1990."
```

Comment:
In previous topic, there was:

            formData: {
                name: "John Heart",
                officeNumber: 901,
                hireDate: new Date(2012, 4, 13)
            }

Two fields are added:
                position: "CEO",
                notes: "John has been in the Audio/Video industry since 1990."

imho one field is better

Reply:
I would like to keep this exact number to make the form look like in our demo (2 columns with even number of items and one item that spans two columns):
![image](https://user-images.githubusercontent.com/33040934/104441068-41b04280-55a4-11eb-8151-4e3addc0a97e.png)